Sources/CombineSchedulers/Internal/Deprecations.swift

```diff
@@ -0,0 +1,83 @@
+#if canImport(Combine)
+  import Combine
+  import Foundation
+
+  // To later deprecate:
+
+  @available(macOS 10.15, iOS 13, tvOS 13, watchOS 6, *)
+  extension Scheduler
+  where
+    SchedulerTimeType == DispatchQueue.SchedulerTimeType,
+    SchedulerOptions == DispatchQueue.SchedulerOptions
+  {
+//    @available(*, deprecated, renamed: "immediate")
```

Comment:
Do you think we should deprecate these in the next release? Or be gentler?

Reply:
Down to deprecate now!